web/generic_scripting.js

```diff
@@ -28,7 +28,7 @@ class GenericScripting {
 
   async createSandbox(data) {
     const sandbox = await this._ready;
-    sandbox.create(data);
+    return sandbox.create(data);
```

Comment:
The `pdf.sandbox.js` file *can* fail to load, if e.g. the option is incorrectly set or the file isn't available, and in that case the `this._ready` promise will be rejected.
That will cause the `if (!success) { ... }` check added in the `web/app.js` file to not be hit, since an error is thrown, and things won't then be clean-up as expected. To avoid this situation, I suggest modifying this method as follows:
```javascript
    try {
      const sandbox = await this._ready;
      return sandbox.create(data);
    } catch (ex) {
      return false;
    }
```